src/assertions/there.js

```diff
@@ -0,0 +1,21 @@
+import elementExists from '../util/element-exists';
+
+export default function there(client, chai, utils) {
```

Comment:
Hmm... I must still be doing it wrong (it might be something with the test setup, which I don't completely understand and mostly copied form the visible tests)?

I've renamed `there` to `exists` and replaced my tests with, for example:
```javascript
    context("When element doesn't exist", () => {
        it('Should throw an error', () => {
            elementExists.throws(new Error());
            expect(() => expect('.some-selector').to.exist).to.throw();
        });
    });
```

But that fails (no error is thrown)... any ideas?